level-ws.js

```diff
@@ -116,11 +116,10 @@ WriteStream.prototype.toString = function () {
   return 'LevelUP.WriteStream'
 }
 
-WriteStream.prototype.destroy = function () {
-  if (this._destroyed) return
+WriteStream.prototype._destroy = function (err, cb) {
   this._buffer = null
-  this._destroyed = true
   this.emit('close')
+  cb(err)
```

Comment:
If there is an `err`, `close`will be emitted before `error`. We can attack that in a separate PR (also to use `this.destroy(err)` instead of `this.emit('error', err)`.